opencr/admin.OCRUser.template.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
Are you changing the default password in the file?

Reply:
I would assume we should change the username to **admin**

Reply:
Okay. Do you know how to generate new password for this user?

Reply:
Well, may be my first question should be, why are you including this file?

Reply:
Because I want to change the default admin user